idemp-base/src/org/bandahealth/idempiere/base/modelevent/ProductModelEvent.java

```diff
@@ -36,11 +36,7 @@ protected void doHandleEvent(Event event) {
 		} else {
 			return;
 		}
-		if (event.getTopic().equals(IEventTopics.PO_BEFORE_NEW)) {
-			beforeSaveRequest(product);
-		} else if (event.getTopic().equals(IEventTopics.PO_BEFORE_CHANGE)) {
-			beforeSaveRequest(product);
-		} else if (event.getTopic().equals(IEventTopics.PO_AFTER_NEW)
+		if (event.getTopic().equals(IEventTopics.PO_AFTER_NEW)
```

Comment:
The tests were creating long product names and, when this logic was running, the `value` was being truncated to a certain number of characters, resulting in duplicates. Since we don't show or use the `value` anywhere in the app, I didn't think it was needed. We also didn't need to go update previous stuff since it's not used anywhere. Thoughts?

Reply:
We can move this discussion to [go-2405](https://issues.bandahealth.org/issue/go-2405/Update-how-value-column-used-on-products) and on discord. I've removed it from this PR (though the tests will be broken until we do something about the other issue.

Reply:
I've undone this change, though can't test without the [go-2405](https://issues.bandahealth.org/issue/go-2405/Update-how-value-column-used-on-products) fix. Will see if the problem re-appears afterwards and I'll create a separate issue to address that problem when/if it occurs.